package.json

```diff
@@ -7,6 +7,7 @@
     "coveralls": "cat ./js_coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./js_coverage"
   },
   "dependencies": {
+    "axios": "^0.15.3",
```

Comment:
We already have jQuery on the project, why not use `$.ajax`? It's compliant with Axios interface.